configure.ac

```diff
@@ -6231,6 +6264,14 @@ fi
 
 if test "x$with_libdpdk" = "xyes"
 then
+  plugin_dpdkevents="no (DPDK version < 16.07)"
```

Comment:
I really dislike version checks. Is there no symbol in DPDK that you can check for existence?